alonzo/impl/src/Cardano/Ledger/Alonzo/Tx.hs

```diff
@@ -161,16 +165,23 @@ import qualified Shelley.Spec.Ledger.UTxO as Shelley
 
 -- ===================================================
 
+{-# DEPRECATED IsValidating "Use IsValid instead" #-}
+type IsValidating = IsValid
+
+{-# DEPRECATED isValidating "Use isValid instead" #-}
+isValidating :: ValidatedTx era -> IsValid
+isValidating = isValid
```

Comment:
For completeness we should also have a deprecated `HasField` instance.

Reply:
great catch, thank you!

Reply:
@nc6 is it possible to deprecate a class instance? It doesn't seem like it to me. I can add the instance, and leave a TODO to remove it when we remove the other deprecated functions.